source/page-pricing.html.erb

```diff
@@ -141,9 +141,9 @@ section_class: 'section_default'
         the [App Store](#{config[:ios_store_url]}){:target='_blank'}"
       ],
       [
-        "Broker Email Imports",
-        :broker_email_import,
-        "Forward your trade confirmations for
+        "Import via Trade Confirmations",
```

Comment:
To match https://test-portfolio.sharesight.com/plans/new via https://github.com/sharesight/investapp/pull/6322 

```suggestion
        "Trade Confirmation Email Imports",
```